src/script/services/manifest.ts

```diff
@@ -13,19 +13,40 @@ import { cleanUrl } from '../utils/url';
 import { setURL } from './app-info';
 
 export const emitter = new EventTarget();
-let manifest: Lazy<Manifest>;
-let maniURL: Lazy<string>;
 
-let generatedManifest: Lazy<Manifest>;
+export let boilerPlateManifest: Manifest = {
+  dir: 'auto',
+  display: 'fullscreen',
+  name: 'placeholder',
+  short_name: 'placeholder',
+  start_url: undefined,
+  scope: '/',
+  lang: 'en',
+  description: 'placeholder description',
+  theme_color: 'none',
+  background_color: 'none',
+  icons: [],
+  screenshots: [],
+};
+
+let manifest: Manifest = boilerPlateManifest;
+let maniURL: Lazy<string>;
+let fetchAttempted = false;
+let generated = false;
```

Comment:
the main thing I did was switch the implementation from having two different manifest objects it works from, two bools, one to track if we've attempted a fetch of the manifest within the section and if we generated it.

Reply:
oh also the default boiler plate being what the manifest looks like, keeps the assigning of this boilerplate out of our components if they fail to fetch